pkg/reconciler/revision/resources/constants.go

```diff
@@ -35,6 +35,9 @@ const (
 
 	// AppLabelKey is the label defining the application's name.
 	AppLabelKey = "app"
+
+	// InitContainerImage is the image used by the init container
+	InitContainerImage = "busybox:1.30.1-musl"
```

Comment:
We should use a simple Go binary we build from scratch instead of this and a script.

Grab me Tuesday if you want the long version with "why?" :)

Reply:
+100 
Including random images makes open source license compliance more difficult

Reply:
And generally making sure those images free from security vulnerabilities